internal/gitinterface/commit.go

```diff
@@ -291,7 +292,7 @@ func (r *Repository) verifyCommitSignature(ctx context.Context, commitID Hash, k
 		}
 
 		return nil
-	case signerverifier.RSAKeyType, signerverifier.ECDSAKeyType, signerverifier.ED25519KeyType:
+	case signerverifier.RSAKeyType, signerverifier.ECDSAKeyType, signerverifier.ED25519KeyType, ssh.SSHKeyType:
```

Comment:
This should eventually drop the explicit RSA, ECDSA, etc types. We want just SSH here.

Reply:
Yeah, I think we should do that too.